### PR TITLE
Linking my blog

### PR DESCRIPTION
**Pull request by hshafaq7**
Please read and understand everything below
**Do not delete any text other than where you are instructed.**

**Students: If one of them is applicable to you. Please check it.**

Check by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [ ] Read and understood (see CONTRIBUTING.md)
- [ ] Included a Preview link and screenshot showning after and before the changes.
- [ ] Images are `240 x 240` [w x h].
- [ ] Included a description of change below.
- [ ] Squashed the commits.

# Changes done in this Pull Request

- If your change will be reflected on the website, please provide a **Test-Link** (**Hint : `gh-pages`**)
- Fixes`#<Issue no.>`

### CODE REVIEW

**Comment by codedsun** [comment]
@hshafaq7 check why the Travis is failing and correct it.

File: _data/blogs.yml [hunk 1]
```diff
@@ -262,3 +262,8 @@
   title: Explore & Discover a new essence in Technology
   img: Saniya blog pic.jpg
   author_name: Saniya Kolangde
+  
+- url:https://mygeekyworld786.wordpress.com/
```
Comment (nikit19) [inline _data/blogs.yml:266]: Please add a space after `:`

**Review by nikit19 — CHANGES_REQUESTED** [review]
You also have to upload an image

File: _data/blogs.yml (hunk 1 above)
Comment (akshatnitd) [reply _data/blogs.yml:266]: Add a space after `:`

File: _data/blogs.yml [hunk 2]
```diff
@@ -262,3 +262,8 @@
...
+  title: Here I come Coding In
+  img: haider2
```
Comment (akshatnitd) [inline _data/blogs.yml:268]: You need to write  the name of the image file that you upload on the `img/blog/` folder. Also, add the image in the said folder.

**Review by akshatnitd — CHANGES_REQUESTED** [review]
Please make the changes

**Comment by harshit98** [comment]
Similar PR #779 merged.

**Pull request closed without merging**